### PR TITLE
Update frontend performance links

### PR DESCRIPTION
**Pull request by DavidBiddle**
This PR:
- Adds links for some of the tools mentioned
- Removes unused links
- fix broken minification link
- updates the `last_reviewed_on` date

As far as I can tell the rest of the content is still current.

### CODE REVIEW

File: source/standards/optimise-frontend-perf.html.md.erb [hunk 1]
```diff
@@ -97,7 +97,7 @@ You can find out more about improving your website’s frontend performance by r
 The Service Manual has more suggestions about [how you can test frontend performance][].
 
 [frontend performance]: https://www.gov.uk/service-manual/technology/how-to-test-frontend-performance
-[minification]: https://minifycode.com/
+[minification]: https://web.dev/reduce-network-payloads-using-text-compression/
```
Comment (domoscargin) [inline source/standards/optimise-frontend-perf.html.md.erb:100]: 👍

File: source/standards/optimise-frontend-perf.html.md.erb [hunk 2]
```diff
@@ -113,9 +113,8 @@ The Service Manual has more suggestions about [how you can test frontend perform
 [Gulp]: https://gulpjs.com/
 [Speedcurve]: https://www.speedcurve.com/
 [Real user monitoring]: https://insidegovuk.blog.gov.uk/2021/06/16/how-real-user-monitoring-will-improve-gov-uk-for-everyone/
-[npm-scripts]: https://css-tricks.com/why-npm-scripts/
 [Continuous Delivery (CD) and Continuous Integration (CI) workflow]: https://www.gov.uk/service-manual/technology/deploying-software-regularly
-[Google PageSpeed]: https://developers.google.com/speed/pagespeed/module/
+[Google PageSpeed Insights]: https://pagespeed.web.dev/
```
Comment (domoscargin) [inline source/standards/optimise-frontend-perf.html.md.erb:117]: 👍

**Review by domoscargin — APPROVED** [review]
Looks good to me - just needs a date bump.

**Comment by DavidBiddle** [comment]
@domoscargin thanks for reviewing! 
Unfortunately force-pushing the date bump has dismissed your review, apologies 😬

**Pull request merged**